src/contrib/google/auth.ts

```diff
@@ -82,13 +82,16 @@
   }
 }
 
-export async function handleRejection(token: string, err: any): Promise<Error> {
-  console.debug("Google rejected request", { err });
-  if (err.result == null) {
-    return err;
+export async function handleRejection(
+  token: string,
+  error: any
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/pixiebrix/pixiebrix-extension/pullRequest?prid=7635933)
```suggestion
  error: unknown
```